core/src/main/java/org/mule/runtime/core/api/streaming/StreamingManager.java

```diff
@@ -103,4 +106,13 @@
    */
   @Deprecated
   void manage(InputStream inputStream, CoreEvent creatorEvent);
+
+  /**
+   * @param provider the provider for which you want the correspondent byte and object providers.
```

Comment:
this jdoc is too vague. Here you should explain that this is for the case in which items in an iterable stream contain nested byte streams.